examples/listbox/listbox-grouped.html

```diff
@@ -0,0 +1,243 @@
+      <li><a href="../../#Listbox">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+  <h1>Listbox Example with Grouped Options</h1>
+  <p>
+    The following example implementation of the <a href="../../#Listbox">design pattern for listbox</a> demonstrates a single-select listbox widget with grouped options.
+    This widget is functionally similar to an HTML <code>select</code> element with <code>size</code> greater than 1 and options grouped into categories with labeled <code>optgroup</code> elements.
+  </p>
+  <p>Similar examples include:</p>
+  <ul>
+    <li><a href="listbox-scrollable.html">Scrollable Listbox Example</a>: Single-select listbox that scrolls to reveal more options, similar to HTML <code>select</code> with <code>size</code> attribute greater than one.</li>
+    <li><a href="listbox-rearrangeable.html">Example Listboxes with Rearrangeable Options</a>: Examples of both single-select and multi-select listboxes with accompanying toolbars where options can be added, moved, and removed.</li>
+    <li><a href="listbox-collapsible.html">Collapsible Dropdown Listbox Example</a>: Single-select collapsible listbox that expands when activated, similar to HTML <code>select</code> with the attribute <code>size=&quot;1&quot;</code>.</li>
+  </ul>
+  <section>
+    <h2 id="ex_label">Example</h2>
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <div id="ex">
+      <div class="listbox-area">
+        <div>
+          <span id="ss_elem" class="listbox-label">Choose your animal sidekick</span>
+          <div id="ss_elem_list" tabindex="0" role="listbox" aria-labelledby="ss_elem">
+            <ul role="group" aria-labelledby="cat1">
+              <li role="presentation" id="cat1">Land</li>
+              <li id="ss_elem_1" role="option">Cat</li>
+              <li id="ss_elem_2" role="option">Dog</li>
+              <li id="ss_elem_3" role="option">Tiger</li>
+              <li id="ss_elem_4" role="option">Reindeer</li>
+              <li id="ss_elem_5" role="option">Raccoon</li>
+            </ul>
+            <ul role="group" aria-labelledby="cat2">
+              <li role="presentation" id="cat2">Water</li>
+              <li id="ss_elem_6" role="option">Dolphin</li>
+              <li id="ss_elem_7" role="option">Flounder</li>
+              <li id="ss_elem_8" role="option">Eel</li>
+            </ul>
+            <ul role="group" aria-labelledby="cat3">
+              <li role="presentation" id="cat3">Air</li>
+              <li id="ss_elem_9" role="option">Falcon</li>
+              <li id="ss_elem_10" role="option">Winged Horse</li>
+              <li id="ss_elem_11" role="option">Owl</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    <h4>Notes</h4>
```

Comment:
The "Notes" heading should be `<h3>` instead of `<h4>` to avoid skipping a level.

Reply:
Fixed, thanks @carmacleod!

Reply:
Wow - thanks for fixing so quickly!